pkgs/development/libraries/pipewire/default.nix

```diff
@@ -116,14 +107,14 @@ stdenv.mkDerivation rec {
 
   FONTCONFIG_FILE = fontsConf; # Fontconfig error: Cannot load default config file
 
-  doCheck = true;
-
-  # Pulseaudio asserts lead to dev references.
-  # TODO This should be fixed in the pulseaudio sources instead.
-  preFixup = ''
-    remove-references-to -t ${libpulseaudio.dev} "$(readlink -f $pulse/lib/libpulse.so)"
+  postInstall = ''
+    moveToOutput "share/systemd/user/pipewire-pulse.*" "$pulse"
+    moveToOutput "lib/systemd/user/pipewire-pulse.*" "$pulse"
+    moveToOutput "bin/pipewire-pulse" "$pulse"
   '';
 
+  doCheck = true;
```

Comment:
nit: this should remain in the same place according to https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/attribute-ordering.md.